baserun/instrumentation/anthropic.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import logging
+from typing import Collection, Any
+
+from openai import Completion
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import (
+    unwrap,
+)
+from opentelemetry.sdk.trace import Span
+from opentelemetry.trace import get_tracer
+from wrapt import wrap_function_wrapper
+
+from baserun.instrumentation.span_attributes import (
+    SpanAttributes,
+    ANTHROPIC_VENDOR_NAME,
+)
+from baserun.instrumentation.wrappers import instrumented_wrapper
+
+logger = logging.getLogger(__name__)
+
+_instruments = ("anthropic >= 0.3.0",)
+__version__ = "0.1.0"
+
+WRAPPED_METHODS = [
+    {
+        "object": "Completions",
+        "method": "create",
+        "span_name": "anthropic.completions.create",
+    },
+    {
+        "object": "AsyncCompletions",
+        "method": "create",
+        "span_name": "async_completions.create",
+    },
+]
+
+
+class AnthropicInstrumentor(BaseInstrumentor):
+    """An instrumentor for OpenAI's client library."""
+
+    def instrumentation_dependencies(self) -> Collection[str]:
+        return _instruments
+
+    def set_request_attributes(self, span: Span, kwargs: dict[str, Any]):
+        span.set_attribute(SpanAttributes.LLM_VENDOR, ANTHROPIC_VENDOR_NAME)
```

Comment:
We still need to set up the various span attributes for anthropic, right?

Reply:
We don't have a ton of request data from anthropic, but I did add model, stop sequences, and max tokens.